### ISSUE TITLE
Send Money from an Intent

### ISSUE THREAD

**Issue by lostdroid** [barmstrong/bitcoin-android#31]
A useful feature may be implement Intent support to allow money to be sent from another application.  The Intent could support the Bitcoin Address to send the money to and optional amount and a memo.  No money would be sent until the user confirms by pressing Send.

**Comment by barmstrong** [comment]
I agree, this would be awesome.  Same for links in browsers following the bitcoin uri format.
